spec/models/opportunity_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe Opportunity do
+  it { should belong_to(:organization) }
+
+  it { should have_valid(:description).when(
+    "Lead an HTML/CSS workshop for middle school girls")
+    }
```

Comment:
`}` at 8, 4 is not aligned with `it { should have_valid(:description).when(` at 6, 2.